addons/account/static/src/components/actionable_errors/actionable_errors.js

```diff
@@ -4,13 +4,26 @@ import { registry } from "@web/core/registry";
 import { Component } from "@odoo/owl";
 import { standardFieldProps } from "@web/views/fields/standard_field_props";
 
+const DISPLAY_ORDER = [
+    'danger',
+    'warning',
+    'info',
+];
+
 export class ActionableErrors extends Component {
     static props = { ...standardFieldProps };
     static template = "account.ActionableErrors";
 
     async handleOnClick(errorData){
         this.env.model.action.doAction(errorData.action);
     }
+
+    get sortedActionableErrors() {
+        let data = this.props.record.data[this.props.name];
```

Comment:
As we're not modifying data after, we should use `const` instead.
```suggestion
        const data = this.props.record.data[this.props.name];
```